applications/dashboard/models/BannerImageModel.php

```diff
@@ -71,6 +71,16 @@ public function renderBanner(array $props = []): \Twig\Markup {
         } else {
             $html = "<div data-react='community-content-banner' data-props='$propsJson'><div style=\"minHeight='500px'\"></div></div>";
         }
+
+        if (Gdn::themeFeatures()->useDataDrivenTheme()) {
+            /** @var \Garden\EventManager $eventManager */
+            $eventManager = Gdn::getContainer()->get(\Garden\EventManager::class);
+            $afterBanner = $eventManager->fire('AfterBanner', $this);
+            if (!empty($afterBanner)) {
+                $html .= implode("", $afterBanner);
+            }
+        }
```

Comment:
```suggestion
        /** @var \Garden\EventManager $eventManager */
        $eventManager = Gdn::getContainer()->get(\Garden\EventManager::class);
        $afterBanner = $eventManager->fire('AfterBanner', $this);
        if (!empty($afterBanner)) {
            $html .= implode("", $afterBanner);
        }
```

We shouldn't need this check here. Only data driven themes call this function, and if something else did call it we probably want to render anyways.